### PR TITLE
 ui, build: reproducible geography build

### PR DESCRIPTION
**Pull request by couchand**
Previously, the geographical data powering our Node Map came from
a blob of uncertain origin.  This replaces that with a blob of
documented and reproducible provenance.  Specifically, we make use
of the us-atlas and world-atlas packages from TopoJSON, which get
the source data from Natural Earth.

Fixes: #24159
Release note: None

Before: 
mercator

After:
equirect

### CODE REVIEW

File: pkg/ui/scripts/topo.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+
+const d3geo = require('d3-geo');
+const d3geoProjection = require('d3-geo-projection');
+const topojson = require('topojson');
+
+const us_orig = require('../node_modules/us-atlas/us/10m.json');
```
Comment (vilterp) [inline pkg/ui/scripts/topo.js]: nit: using underscores in a JS file — why a different style for Node?

File: pkg/ui/scripts/topo.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+const invert = d3geo.geoTransform({
+    point: function(x, y) {
+        const inverted = projection.invert([x, y]);
+        this.stream.point(inverted[0], inverted[1]);
+    }
+});
+
+const us_feat_projected = topojson.feature(us_orig, {
+    type: "GeometryCollection",
+    geometries: us_orig.objects.states.geometries,//.filter(s => s.id != "02"),
```
Comment (vilterp) [inline pkg/ui/scripts/topo.js]: nit: leftover filter

File: pkg/ui/ccl/src/views/clusterviz/containers/map/worldmap.tsx [hunk 3]
```diff
@@ -33,15 +32,6 @@ export class WorldMap extends React.Component<WorldMapProps> {
             />,
           )}
         </g>
-        <g>
-          {usStateShapes.features.map((feature: any, i: number) =>
```
Comment (vilterp) [inline pkg/ui/ccl/src/views/clusterviz/containers/map/worldmap.tsx]: no more US states? These do seem kind of helpful.

File: pkg/ui/package.json [hunk 4]
```diff
@@ -104,13 +105,16 @@
     "stylus": "^0.54.5",
     "stylus-loader": "^3.0.1",
     "thread-loader": "^1.1.1",
+    "topojson": "^3.0.2",
     "ts-loader": "^2.3.4",
     "tslint": "^5.10.0",
     "typescript": "^2.2.1",
     "url-loader": "^0.5.8",
+    "us-atlas": "^1.0.2",
```
Comment (vilterp) [inline pkg/ui/package.json]: Nice that these are available as node modules. If we're not doing US states, remove this one (but I prefer to do US states I think)

**Review by vilterp — COMMENTED** [review]
Nice. Just a coupla nits.

**Review by couchand — COMMENTED** [review]
> no more US states? These do seem kind of helpful.

If you look carefully at the script in `topo.js` (which you can be forgiven for not doing because it hurts to read), you can see that I'm combining the US state outlines and country outlines into a single file.  I did this for two reasons: one, there's no reason to have the US national outline twice, and two, it allows us to simplify the geography consistently.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/32617#-:-LSLEjMP14R81zB8DlGR:b-rqsnxy)* status: :shipit: complete! 0 of 0 LGTMs obtained

___
*[pkg/ui/package.json, line 113 at r3](https://reviewable.io/reviews/cockroachdb/cockroach/32617#-LSH5YBg7WTgBHA9TENy-r3-113:-LSLEAEH1h-QM00eWPn2:bisio28) ([raw file](https://github.com/cockroachdb/cockroach/blob/24c37c218bf976a0777c6092d359cc086b7ea419/pkg/ui/package.json#L113)):*
<details><summary><i>Previously, vilterp (Pete Vilter) wrote…</i></summary><blockquote>

Nice that these are available as node modules. If we're not doing US states, remove this one (but I prefer to do US states I think)
</blockquote></details>

see other.
___
*[pkg/ui/scripts/topo.js, line 10 at r3](https://reviewable.io/reviews/cockroachdb/cockroach/32617#-LSH5YBkzQ5fjm9B3gZw-r3-10:-LSLEDcp2lnEIeuiUm4c:bu47j3) ([raw file](https://github.com/cockroachdb/cockroach/blob/24c37c218bf976a0777c6092d359cc086b7ea419/pkg/ui/scripts/topo.js#L10)):*
<details><summary><i>Previously, vilterp (Pete Vilter) wrote…</i></summary><blockquote>

nit: using underscores in a JS file — why a different style for Node?
</blockquote></details>

I think if you plotted the number of times I do this, you'd notice I make this mistake particularly frequently on Mondays.

Fixed.
___
*[pkg/ui/scripts/topo.js, line 23 at r3](https://reviewable.io/reviews/cockroachdb/cockroach/32617#-LSH5YBkzQ5fjm9B3gZw-r3-23:-LSLEU2g2B6ZzSc0bD7B:b-87j1qu) ([raw file](https://github.com/cockroachdb/cockroach/blob/24c37c218bf976a0777c6092d359cc086b7ea419/pkg/ui/scripts/topo.js#L23)):*
<details><summary><i>Previously, vilterp (Pete Vilter) wrote…</i></summary><blockquote>

nit: leftover filter
</blockquote></details>

Gone.

**Review by vilterp — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/32617)* status: :shipit: complete! 0 of 0 LGTMs obtained

___
*[pkg/ui/scripts/topo.js, line 10 at r3](https://reviewable.io/reviews/cockroachdb/cockroach/32617#-LSH5YBkzQ5fjm9B3gZw-r3-10:-LSLHMBD9NbPoPdm_8aD:b-m7efrc) ([raw file](https://github.com/cockroachdb/cockroach/blob/24c37c218bf976a0777c6092d359cc086b7ea419/pkg/ui/scripts/topo.js#L10)):*
<details><summary><i>Previously, couchand (Andrew Couch) wrote…</i></summary><blockquote>

I think if you plotted the number of times I do this, you'd notice I make this mistake particularly frequently on Mondays.

Fixed.
</blockquote></details>

Writing a bunch of Rust on the weekends? :P

**Pull request merged**